packages/saved-views-react/src/api/utilities/translation/viewExtractorSavedViewToLegacySavedView.ts

```diff
@@ -286,16 +286,11 @@ export function savedViewITwin3dToLegacy3dSavedView(
   return legacyView;
 }
 
-/**
- * append Hidden Categories Or Models To Legacy Saved View
- * @param iTwinView new schema
- * @param legacyView
- * @returns iModelViewData
- */
+/** Append Hidden Categories Or Models To Legacy Saved View. */
 function appendHiddenCategoriesToLegacyView(
-  iTwinView: SavedViewBase,
+  iTwinView: ViewITwin2d | ViewITwin3d,
```

Comment:
Why drop saved view in param name and type for parm but keep legacy**Saved**View instead of legacyView.

Reply:
Do we want **Saved** to be in view names for types and params

Reply:
This code and types existed before the PR, I didn't make any decisions regarding keeping or dropping the word "saved".